task_processing/plugins/mesos/__init__.py

```diff
@@ -8,6 +9,7 @@
 
 def register_plugin(registry):
     return registry \
+        .register_task_executor('logging', LoggingExecutor) \
```

Comment:
Can we call it the 'MesosLoggingExecutor' to try reset expectations about what this really is?

Reply:
Will fix.